src/app/home/avatar-generator/selector/atribute-selector.html

```diff
@@ -0,0 +1,32 @@
+<!-- Rounded dialog -->
```

Comment:
Please add the "component" suffix to the name of this file: `attribute-selector.component.html`